<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Timer</class>
 <widget class="QWidget" name="Timer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="TimerStart">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="TimerPause">
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="TimerReset">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>320</width>
     <height>320</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="PomodoroStats">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>181</width>
     <height>361</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>521</width>
     <height>121</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>519</width>
      <height>119</height>
     </rect>
    </property>
    <widget class="QTableWidget" name="SessionHistoryTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>521</width>
       <height>121</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Start Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>End Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Focus Duration</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Breaks Taken</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
